crates/markdown_preview/src/markdown_parser.rs

```diff
@@ -224,8 +223,54 @@ impl<'a> MarkdownParser<'a> {
 
                 Event::Text(t) => {
                     text.push_str(t.as_ref());
-
                     let mut style = MarkdownHighlightStyle::default();
+                    if let Some(image) = image.clone() {
+                        let is_valid_image = match image.clone() {
+                            Image::Path {
+                                source_range: _,
+                                display_path,
+                                path: _,
+                                link: _,
+                            } => gpui::ImageSource::try_from(display_path).is_ok(),
+                            Image::Web {
+                                source_range: _,
+                                url,
+                                link: _,
+                            } => match isahc::get(url) {
```

Comment:
I think we should just skip this check. It's ok if the image is invalid on the web, just let GPUI fail to render it.